### PR TITLE
Handle missing in GapEncoder

### PR DESCRIPTION
**Pull request by LilianBoulard**
Similar to #128

Add missing values handling in the GapEncoder.
Required for #178

### CODE REVIEW

File: dirty_cat/gap_encoder.py [hunk 1]
```diff
@@ -514,6 +515,14 @@ class GapEncoder(BaseEstimator, TransformerMixin):
     max_iter_e_step : int, default=20
         Maximum number of iterations to adjust the activations h at each step.
 
+    handle_missing : 'error' or '' (default)
+        Whether to raise an error or impute with blank string '' if missing
```
Comment (GaelVaroquaux) [inline dirty_cat/gap_encoder.py:519]: Maybe "passthrough" instead of ''

**Comment by GaelVaroquaux** [comment]
I made a little comment, but aside from this commit, this PR is good for me. Thanks!!

**Review by alexis-cvetkov — APPROVED** [review]

File: dirty_cat/gap_encoder.py (hunk 1 above)
Comment (LilianBoulard) [reply dirty_cat/gap_encoder.py:519]: Well I guess we're not passing through, as we're imputing the missing data. Am I missing something ?
Thanks :)

**Comment by GaelVaroquaux** [comment]
> Well I guess we're not passing through, as we're imputing the missing data. Am
> I missing something ?

Point taken, I'll remove this.

File: dirty_cat/gap_encoder.py (hunk 1 above)
Comment (GaelVaroquaux) [reply dirty_cat/gap_encoder.py:519]: Maybe "zeroout"?

I just don't love the empty string. It doesn't really tell what it does to a casual reader.

Comment (alexis-cvetkov) [reply dirty_cat/gap_encoder.py:519]: I agree the empty string is not clear, maybe we can simply set `handle_missing = "impute"` ?

Comment (GaelVaroquaux) [reply dirty_cat/gap_encoder.py:519]: OK, then "zero_impute". I don't like "impute", because people typically expect conditional imputations.

Comment (LilianBoulard) [reply dirty_cat/gap_encoder.py:519]: I changed the default parameter to `zero_impute`, while still imputing with empty strings

**Comment by LilianBoulard** [comment]
Merging !

**Comment by GaelVaroquaux** [comment]
Thanks!! Sorry, I'm teaching all day. Little feedback today

**Pull request merged**